### PR TITLE
[NoJira] add theming to links

### PR DESCRIPTION
**Pull request by matteo-hertel**

### CODE REVIEW

File: packages/bpk-component-theme-toggle/src/theming.js [hunk 1]
```diff
@@ -50,6 +50,12 @@ const bpkCustomTheme = {
   buttonSecondaryHoverBackgroundColor: '#ffffff',
   buttonSecondaryActiveBackgroundColor: '#ffffff',
 
+  linkColor: '#865f9e',
+  linkTextColor: '#865f9e',
+  linkHoverColor: '#461962',
+  linkActiveColor: '#2d244c',
+  linkVisitedColor: '#1a1331',
+
```
Comment (shaundon) [inline packages/bpk-component-theme-toggle/src/theming.js]: As the PR for refactoring this file to use variables has been merged, can you update this to use them too?

**Comment by matthewdavidson** [comment]
@matteo-hertel I rebased this for you ;) should be good to go now.

**Comment by matteo-hertel** [comment]
I think the Readme was wrong and I've used that as source for the themable properties, thanks for spotting the issue 😄

File: packages/bpk-component-link/readme.md [hunk 2]
```diff
@@ -46,7 +46,10 @@ export default () => (
 
 For both BpkLink and BpkLinkButton the following theme attributes are required.
 
-* `linkTextColor`
+* `linkColor`
 * `linkHoverColor`
 * `linkActiveColor`
+* `linkColor`
```
Comment (matteo-hertel) [inline packages/bpk-component-link/readme.md]: this does not look right, wrong rebase 😄

Comment (k0nserv) [reply packages/bpk-component-link/readme.md]: Yeah `linkTextColor` is not correct that seems to have been an error on my part, but you fixed that it seems. Just remove the dupes and we are good

**Comment by matteo-hertel** [comment]
Should be all good now!

**Pull request merged**